pkg/controller/tenant/tenant_controller.go

```diff
@@ -267,6 +280,15 @@ func (tc *TenantController) createDefaultNetworkObject(tenantName string) (error
 func (tc *TenantController) createNamespaces(tenant string) (error, bool) {
 	failures := []error{}
 	for _, nsName := range tenantDefaultNamespaces {
```

Comment:
Need to check the tenant's status is active before creating. Otherwise we might run into weird race condition issue when deleting a tenant. 

You can refer to the following logic in createDefaultNetworkObject():
      if tenant.Status.Phase == v1.TenantTerminating {
		klog.Infof("Tenant %q is terminating; skipped the creation of default network", tenantName)
	}